robolectric-shadows/shadows-core/src/main/resources/org/robolectric/shadows/ShadowBackgroundThread.java.vm

```diff
@@ -9,7 +9,7 @@ import org.robolectric.annotation.Resetter;
 import org.robolectric.internal.Shadow;
 import org.robolectric.util.ReflectionHelpers;
 
-@Implements(value = BackgroundThread.class, inheritImplementationMethods = true)
+@Implements(value = BackgroundThread.class, isInAndroidSdk = false, inheritImplementationMethods = true)
```

Comment:
I'm sorry that slipped out of the original PR. I did have that in a version I was working on. Thanks for catching this!